letsencrypt/cli.py

```diff
@@ -541,7 +553,12 @@ def create_parser(plugins, args):
 
     _create_subparsers(helpful)
 
-    return helpful.parser
+    return helpful.parser, helpful.args
+
+# For now unfortunately this constant just needs to match the code below;
+# there isn't an elegant way to autogenerate it in time.
+VERBS = ["run", "auth", "install", "revoke", "rollback", "config_changes",\
```

Comment:
- please don't use the the backslash; this should've been caught by the pylint...
- how come you're not able to autogenerate it? just go through the registered subparsers...